### PR TITLE
Add option to `mc admin policy info` to save policy JSON to file

### PR DESCRIPTION
**Pull request by donatello**

### CODE REVIEW

**Review by harshavardhana — APPROVED** [review]

File: cmd/admin-policy-info.go [hunk 1]
```diff
@@ -93,6 +105,18 @@ func mainAdminPolicyInfo(ctx *cli.Context) error {
...
+	if policyFile != "" {
+		f, err := os.Create(policyFile)
+		if err != nil {
+			fatalIf(probe.NewError(err).Trace(args...), "Could not open given policy file")
+		}
+		_, err = f.Write(pinfo.Policy)
+		if err != nil {
+			fatalIf(probe.NewError(err).Trace(args...), "Could not write to given policy file")
+		}
+	}
```
Comment (harshavardhana) [inline cmd/admin-policy-info.go:118]: can you support `-` to write to os.Stdout? @donatello

**Review by harshavardhana — CHANGES_REQUESTED** [review]

File: cmd/admin-policy-info.go [hunk 2]
```diff
@@ -18,25 +18,34 @@
...
 	"github.com/fatih/color"
 	"github.com/minio/cli"
 	"github.com/minio/madmin-go"
 	"github.com/minio/mc/pkg/probe"
 	"github.com/minio/pkg/console"
 )
 
+var policyInfoFlags = []cli.Flag{
+	cli.StringFlag{
+		Name:  "policy-file, f",
```
Comment (vadmeste) [inline cmd/admin-policy-info.go:32]: `output-file, o` or `save, s` are better in my opinion.

File: cmd/admin-policy-info.go [hunk 3]
```diff
@@ -18,25 +18,34 @@
...
+		Usage: "additionally (over-)write policy JSON to given file",
```
Comment (vadmeste) [inline cmd/admin-policy-info.go:33]: I think maybe let's avoid printing in the stdout when this flag is passed

**Review by vadmeste — COMMENTED** [review]
Some comments

File: cmd/admin-policy-info.go (hunk 2 above)
Comment (donatello) [reply cmd/admin-policy-info.go:32]: This is an option to save just policy definition, not the metadata that is output to stdout

File: cmd/admin-policy-info.go (hunk 3 above)
Comment (harshavardhana) [reply cmd/admin-policy-info.go:33]: @donatello ^^

**Review by harshavardhana — APPROVED** [review]

**Review by poornas — APPROVED** [review]

**Pull request merged**